wandb/sdk/lib/retry.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args: Any, **kwargs: Any) -> _R:  # noqa: C901
         if num_retries is None:
             num_retries = 1000000
 
-        if os.environ.get("WANDB_TEST"):
+        if os.environ.get("DISABLE_RETRY"):
```

Comment:
or better yet, maybe add this to the env.py file and get the value from that module?

```suggestion
        if os.environ.get("WANDB_DISABLE_RETRY"):
```